### DISCUSSION TITLE
Inactive tab: blank page

### DISCUSSION THREAD

**Discussion by RadovanPelka** [vercel/next.js#18975] [Help]
Hi guys,

I have a problem. This will happen after 15 minutes approx: 

Steps:
 1) Browser (tab 1): nextjs app
 2) (tab 2): for example: google.com (after 15m)
 3) ( tab 1): blank page see
DOM:
![image](https://user-images.githubusercontent.com/29738200/98536726-8c51cd80-2288-11eb-8812-27bf5a62df78.png)

Could you please help me?

![image](https://user-images.githubusercontent.com/29738200/98535492-7d6a1b80-2286-11eb-938e-d90285721913.png)

**Comment by ChimingPhang** [comment]
Have you solved it?
